.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - '**'
+
+jobs:
+  build-docker:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: wpilib/roborio-cross-ubuntu:2020-18.04
+            artifact-name: Athena
+            build-options: ""
```

Comment:
Curious as to why there isn't a "-Ponlylinuxathena" here. Looks like the job is building both Linux x86-64 artifacts and roboRIO artifacts.

Reply:
Thank you!